app.js

```diff
@@ -301,9 +301,11 @@ $(document).ready(function () {
 
     APP.translation.init(settings.getLanguage());
 
-    APP.API.init(APP.tokenData.externalAPISettings);
-    // For remote control testing:
-    // APP.API.init({forceEnable: true});
+    APP.API.init(APP.tokenData.jwt ? {
```

Comment:
Now I see that it was moved from TokenData.js. Still some comment would be helpful. I wonder how is the change related to remote control.